src/api/app/views/webui2/webui/request/show.html.haml

```diff
@@ -0,0 +1,68 @@
+:ruby
+  @pagetitle = "Request #{@bs_request.number}"
+  @layouttype = 'custom'
+
+.card.mb-3
+  = render partial: 'tabs', locals: { request: @bs_request }
+  .card-body
+    %h3
+      Request #{@bs_request.number} (#{@bs_request.state})
```

Comment:
you will probably have to adapt this code after the current deliverable

Reply:
Sure